app/scripts/lib/ppom/ppom-middleware.ts

```diff
@@ -122,6 +115,7 @@ export function createPPOMMiddleware<
             request: req,
             securityAlertId,
             chainId,
+            updateSecurityAlertResponse,
           }).then((securityAlertResponse) => {
             updateSecurityAlertResponse(
```

Comment:
As we're passing the update callback to the util, can we omit this `then` entirely plus the one in `transaction/util` since the util can own all the updates?